drag_and_drop_v2/templates/html/js_templates.html

```diff
@@ -113,12 +113,20 @@
                    value="{{ height }}" />
         </div>
         <div class="row">
-          <label for="item-{{id}}-numerical-value">{{i18n "Optional numerical value"}}</label>
-          <input type="text"
+          <label for="item-{{id}}-numerical-value">
+            {{i18n "Optional numerical value (if you set this, students will be prompted for this value after dropping this item)"}}
+          </label>
+          <input type="number"
+                 step="0.1"
                  id="item-{{id}}-numerical-value"
                  class="item-numerical-value" value="{{ numericalValue }}" />
-          <label for="item-{{id}}-numerical-margin">{{i18n "Margin ±"}}</label>
-          <input type="text"
+        </div>
+        <div class="row">
+          <label for="item-{{id}}-numerical-margin">
+            {{i18n "Margin ± (to be considered correct, value entered by user must not differ from expected value by more than this)"}}
```

Comment:
This need to be crystal-clear that this value is only used if the "numerical value" is set. How about "when a numerical value is required, the value entered by user must not differ from expected value by more than this margin; default is zero"